server/src/controllers/orderController.ts

```diff
@@ -45,6 +44,42 @@ orderController.post(
     },
 );
 
+orderController.use(express.json());
+orderController.post(
+    '/webhook',
+    express.raw({type: 'application/json'}),
+    async (req: Request, res: Response) => {
+      const sig = req.headers['stripe-signature'];
+      try {
+        const event = await stripe.webhooks.constructEvent(
+            req.body,
+            sig,
+            webhookKey,
+        );
+
+        const object = event.data.object;
+        const metaData = object.metadata;
+
+        if (metaData.sessionType === '__ticketing') {
+          await ticketingWebhook(
+              prisma,
+              event.type,
+              object.payment_intent,
+              object.id,
+          );
+        // handle donation amount in metadata (will be a donation team task?)
+        } else if (metaData.sessionType === '__donation') {
+        // donation is handled
+        }
```

Comment:
This should be brought up with the donation team

Reply:
I talked to them a couple weeks ago. Both teams have to use the same webhook api callback so the sessionType is being used to determine how to direct control flow between for donation or ticket transaction.

Reply:
Ok, thanks!